go/pkg/clickhouse/schema/001_key_verifications_raw_v2.sql

```diff
@@ -0,0 +1,66 @@
+CREATE TABLE key_verifications_raw_v2
+(
+-- the api request id, so we can correlate the verification with traces and logs
+  request_id String,
+
+  -- unix milli
+  time Int64 CODEC(Delta, LZ4),
+
+  workspace_id String,
+  key_space_id String,
+  -- Empty string if the key has no identity
+  identity_id String,
+  key_id String,
+
+  -- Right now this is a 3 character airport code, but when we move to aws,
+  -- this will be the region code such as `us-east-1`
+  region LowCardinality(String),
+
+  -- Examples:
+  -- - "VALID"
+  -- - "RATE_LIMITED"
+  -- - "EXPIRED"
+  -- - "DISABLED
+  outcome LowCardinality(String),
+
+  tags Array(String) DEFAULT [],
+
+  -- The number of credits spent on this verification
+  -- 0 means no credits were spent
+  spent_credits Int64,
+
+  -- Latency in milliseconds for this verification
+  latency Float64,
+
+  INDEX idx_request_id (request_id) TYPE bloom_filter GRANULARITY 1,
+  INDEX idx_identity_id (identity_id) TYPE bloom_filter GRANULARITY 1,
+  INDEX idx_key_id (key_id) TYPE bloom_filter GRANULARITY 1,
+  INDEX idx_tags (tags) TYPE bloom_filter GRANULARITY 1
+)
+ENGINE = MergeTree()
+PARTITION BY toYYYYMMDD(fromUnixTimestamp64Milli(time))
+ORDER BY (workspace_id, time, key_space_id, outcome)
+TTL toDateTime(fromUnixTimestamp64Milli(time)) + INTERVAL 1 MONTH DELETE
```

Comment:
Should this not be INTERVAL 3 MONTH DELETE 

Our retention is 90 days, Unless I am dumb. Which is very possible and I don't understand this as much as I thought.

Reply:
![CleanShot 2025-09-10 at 08.55.57@2x.png](https://app.graphite.dev/user-attachments/assets/be09dc03-cc73-41c2-b708-145774891370.png)

according to our pricing page its 30